README.md

```diff
@@ -5,48 +5,49 @@
 <a href="https://github.com/camaraproject/QualityOnDemand" title="Repo Size"><img src="https://img.shields.io/github/repo-size/camaraproject/QualityOnDemand?style=plastic"></a>
 <a href="https://github.com/camaraproject/QualityOnDemand/blob/main/documentation/LICENSE.APACHE2.0" title="License"><img src="https://img.shields.io/badge/License-Apache%202.0-green.svg?style=plastic"></a>
 <a href="https://github.com/camaraproject/QualityOnDemand/releases/latest" title="Latest Release"><img src="https://img.shields.io/github/release/camaraproject/QualityOnDemand?style=plastic"></a>
+<a href="https://github.com/camaraproject/Governance/blob/main/ProjectStructureAndRoles.md" title="Incubating API Repository"><img src="https://img.shields.io/badge/Incubating%20API%20Repository-green?style=plastic"></a>
 
 # QualityOnDemand
 
 Repository to describe, develop, document and test the QualityOnDemand APIs
+Incubating API Repository to evolve and maintain the definitions and documentation of QualityOnDemand Service API(s) within the Sub Project [Quality On Demand](https://lf-camaraproject.atlassian.net/wiki/x/hAClB)
 
 ## Scope
 
 * Service APIs for “Quality on Demand” (see APIBacklog.md)
-* It provides the API consumer with the ability to:  
+* The Service APIs provide the API consumer with the ability to:  
   * set quality for a flow within an access network connections (e.g. mobile device connection or fixed access between a home gateway and the service providers gateway router)
-      * Session mode, for a specific duration
-      * Provision mode, indefinitely for each time the device connects to the same access network
+    * Session mode, for a specific duration
+    * Provision mode, indefinitely for each time the device connects to the same access network
   * get notification if the network cannot fulfill  
 * Describe, develop, document and test the APIs (with 1–2 Service Providers)  
 * Started: October 2021
+* Incubating stage since: February 2025
 
 ## Release Information
 
 * Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest public release**.
 
 
-* **The latest public release is [r1.3](https://github.com/camaraproject/QualityOnDemand/tree/r1.3), a patch release of r1.2, with the following API definitions:**
+* **The latest public release is [r2.2](https://github.com/camaraproject/QualityOnDemand/releases/tag/r2.2), with the following API versions:**
 
-  * **quality-on-demand v0.11.1**  
+  * **quality-on-demand v1.0.0 (first stable version)**  
```

Comment:
The links below should be updated to r2.2

Reply:
Upps ... corrected in https://github.com/camaraproject/QualityOnDemand/pull/427/commits/c39a2d9f46f239e59abb63f977eb8be70c1b8ae6